src/rgw/rgw_admin.cc

```diff
@@ -3445,6 +3456,9 @@ int main(int argc, const char **argv)
   std::optional<int> bucket_index_max_shards;
 
   int max_concurrent_ios = 32;
+  int min_age = 1;
```

Comment:
please use a std::chrono type to keep track of units. `ceph::timespan` is a good default for a duration type

what do you think about calling the command option something like `--min-age-hours`?